From 45568625feacf13f1450da6b8cec17bf97c5a3fd Mon Sep 17 00:00:00 2001
From: chenjh <chenjh@rock-chips.com>
Date: Thu, 16 Mar 2017 10:15:34 +0800
Subject: [PATCH] firmware: Kconfig: ROCKCHIP_SIP depends on HAVE_ARM_SMCCC and
 ARCH_ROCKCHIP

HAVE_ARM_SMCCC is default selected by ARM(if CPU_V7) or ARM64

Change-Id: I4bc64d4c98de5fad3179b3121b0f361d6337732c
Signed-off-by: chenjh <chenjh@rock-chips.com>
---
 drivers/firmware/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/firmware/Kconfig b/drivers/firmware/Kconfig
index 1067fe907d9b..4fafa0b2662e 100644
--- a/drivers/firmware/Kconfig
+++ b/drivers/firmware/Kconfig
@@ -178,7 +178,7 @@ config HAVE_ARM_SMCCC
 
 config ROCKCHIP_SIP
 	bool "Rockchip SIP interface"
-	depends on ARM64 && ARM_PSCI_FW
+	depends on HAVE_ARM_SMCCC && ARCH_ROCKCHIP
 	help
 	  Say Y here if you want to enable SIP callbacks for Rockchip platforms
 	  This option enables support for communicating with the ATF.
-- 
2.35.3

